demo/pages/OIO/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { PureComponent } from 'react'
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter'
+import { duotoneForest } from 'react-syntax-highlighter/dist/styles/prism'
+import style from '../../style.less'
+
+const codeSnippet1 = `
+// All OIO components must be wrapped in an OIO Container
+const myComponent = () => (
+   <OIO>
+      <Text>
+         Hello World
+      </Text>
+   </OIO>
+)`
+
+const apiOptions = [{
+   name: 'className',
+   type: 'String',
+   default: '-',
+   description: 'Pass a <code>className</code> to customize className of the OIO Container'
```

Comment:
Just curious, in the descriptions, what is specifically it that's being put in `code` tags? I ask, because `className` is a string prop, but then on line 35, `object` is a javascript type.

Reply:
Good point. I will only use code tags for props and prop values.